README.md

```diff
@@ -97,5 +98,5 @@ for ( console.log('Bob'); (console.log('Curly') === undefined); console('Mo') )
 console.log(i)
 ```
 
--- YOUR ANSWER HERE --
 
+Mo
```

Comment:
Bob
Curly
1
Mo
Curly
2
2

Please experiment with for loops like Carlos did in class.